### ISSUE TITLE
vue3 support

### ISSUE THREAD

**Issue by GHChrisSu** [JohMun/vue-tags-input#153]
please support vue3, thanks

**Comment by tobiasdiez** [comment]
I agree, Vue 3 support would be nice. I've found the following fork for Vue 3 https://github.com/sipec/vue-tags-input but I'm not sure how @sipec plans to forward with this.

**Comment by bistory** [comment]
> I agree, Vue 3 support would be nice. I've found the following fork for Vue 3 https://github.com/sipec/vue-tags-input but I'm not sure how @sipec plans to forward with this.

Unfortunately, @sipec fork doesn't seem to be maintained anymore (1 year without commit, several issues opened with no reply)

**Comment by sipec** [comment]
yeah sorry I've kinda abandoned this to work on other things

**Comment by VojtechLanka** [comment]
I needed this plugin for one project I am working on so I used the code from @sipec as base and created an NPM package. Since I could not get it to build without errors using the webpack I replaced it with Vite. First time I done that but at first glance everything seem to work. Additionally one of the requirements for my project was drag and drop feature for the tags. It is an optional feature that has to be tuned on otherwise it should be identical to the original.

https://www.npmjs.com/package/@vojtechlanka/vue-tags-input

**Comment by NiloCK** [comment]
To be a little more explicit than a ❤ :

I had a zero-fuss drop in replacement success with @VojtechLanka's fork.

Actually - one tiny change - the export seems to no longer be default - so changing your import to `import { VueTagsInput } from '@vojtechlanka/vue-tags-input';` rather than `import VueTagsInput from @johmun/vue-tags-input;`
